src/Avalonia.Build.Tasks/Avalonia.Build.Tasks.csproj

```diff
@@ -105,8 +105,8 @@
       <Compile Include="..\Avalonia.Base\Metadata\NullableAttributes.cs" Link="NullableAttributes.cs" />
       <Compile Remove="../Markup/Avalonia.Markup.Xaml.Loader\xamlil.github\**\obj\**\*.cs" />
       <Compile Remove="../Markup/Avalonia.Markup.Xaml.Loader\xamlil.github\src\XamlX\IL\SreTypeSystem.cs" />
-      <PackageReference Include="Mono.Cecil" Version="0.11.2" />
-      <PackageReference Include="Microsoft.Build.Framework" Version="15.1.548" PrivateAssets="All" />
```

Comment:
This change breaks compatibility with older SDK and Visual Studio versions

Reply:
VS2017 uses MSBuild 15, VS2019 uses MSBuild 16
.NET Core 3.1 SDK (which is LTS and is still supported) uses MSBuild 16

I dont see any particular reason why we should break compatibility with those.

Reply:
Adjusted

Reply:
Make sure that ILRepack picks the correct version